yt/frontends/ramses/field_handlers.py

```diff
@@ -444,12 +444,17 @@ def read_rhs(cast):
             rheader[p.strip()] = cast(v)
 
         with open(fname, 'r') as f:
+            # Read nRTvar, nions, ngroups, iions
             for i in range(4): read_rhs(int)
             f.readline()
+            # Read X and Y fractions
             for i in range(2): read_rhs(float)
             f.readline()
-            for i in range(3): read_rhs(float)
+            # Reat unit_np, unit_pfd, rt_c_frac
+            for i in range(2): read_rhs(float)
```

Comment:
Can we get an updated comment here?  Others might catch it, but it took me a second to realize that the special case was for `rt_c_frac` being multiple items on a line.

Reply:
Sure.

Reply:
also minor nit, can you put the calls to `read_rhs` in their own blocks, like this:

```
for i in range(2): 
    read_rhs(float)
```

I know it was already like that, formatting with explicit carriage returns and tabs makes it clearer there's a new block there.